src/components/Reports/CampaignsHistory/CampaignsHistory.js

```diff
@@ -77,65 +77,64 @@ const CampaignsHistory = ({ location, dependencies: { dopplerApiClient } }) => {
-              className={getSubscriberStatusCssClassName(state.subscriber.status)}
-            ></span>
-            <FormattedMessage id={'subscriber.status.' + state.subscriber.status} />
-          </p>
-        </div>
-        <div className="col-sm-12 dp-block-wlp m-b-36">
-          <div className="dp-table-responsive">
-            <table
-              className="dp-c-table"
-              aria-label="Resultado de historial de suscriptores"
-              summary="Resultado de historial de suscriptores"
-            >
-              <thead>
-                <tr>
-                  <th scope="col">
-                    <FormattedMessage id="subscriber_history_sent_campaigns.grid_campaign" />
-                  </th>
-                  <th scope="col">
-                    <FormattedMessage id="subscriber_history_sent_campaigns.grid_subject" />
-                  </th>
-                  <th scope="col">
-                    <FormattedMessage id="subscriber_history_sent_campaigns.grid_delivery" />
-                  </th>
-                  <th scope="col">
-                    <FormattedMessage id="subscriber_history_sent_campaigns.grid_clicks" />
-                  </th>
-                </tr>
-              </thead>
-              <tbody>
-                {state.sentCampaigns.items.length ? (
-                  state.sentCampaigns.items.map((campaign, index) => (
-                    <tr key={index}>
-                      <td>{campaign.campaignName}</td>
-                      <td>{campaign.campaignSubject}</td>
-                      <td>{campaign.deliveryStatus}</td>
-                      <td>{campaign.clicksCount}</td>
-                    </tr>
-                  ))
-                ) : (
-                  <p className="dp-boxshadow--usermsg bounceIn">
-                    <FormattedMessage id="common.empty_data" />
-                  </p>
-                )}
-                {}
-              </tbody>
-            </table>
+        <div className="dp-box-shadow m-t-36">
+          <div className="col-sm-12 m-t-24">
+            <h2>
+              {state.subscriber.email}
+              {/*TODO implementation {state.subscriber.score} */}
+            </h2>
+            <p>
+              {state.subscriber.firstName ? state.subscriber.firstName.value : ''}{' '}
+              {state.subscriber.lastName ? state.subscriber.lastName.value : ''}
+            </p>
+            <span>
+              {/* the style it's temporal because there is a bug in the styles */}
+              <span className={getSubscriberStatusCssClassName(state.subscriber.status)}></span>
+              <FormattedMessage id={'subscriber.status.' + state.subscriber.status} />
+            </span>
+          </div>
+          <div className="col-sm-12 dp-block-wlp">
+            <div className="dp-table-responsive">
+              <table
+                className="dp-c-table"
+                aria-label="Resultado de historial de suscriptores"
```

Comment:
I would love this texts were internationalized. What do you think? It's a bit beyond the scope of this change, perhaps in another commit.